src/concrete/ml/sklearn/qnn_module.py

```diff
@@ -27,13 +28,15 @@ def __init__(
         n_layers: int,
         n_outputs: int,
         n_hidden_neurons_multiplier: int = 4,
-        n_w_bits: int = 3,
-        n_a_bits: int = 3,
-        n_accum_bits: int = MAX_BITWIDTH_BACKWARD_COMPATIBLE,
+        n_w_bits: int = 4,
+        n_a_bits: int = 4,
+        # No pruning by default as roundPBS keeps the PBS precision low
+        n_accum_bits: int = 32,
         n_prune_neurons_percentage: float = 0.0,
         activation_function: Type = nn.ReLU,
         quant_narrow: bool = False,
         quant_signed: bool = True,
+        power_of_two_scaling: bool = True,  # Default to true: use roundPBS to speed up the NNs
```

Comment:
It's a bit weird here how rounding is implicitly contained in the PoT feature. Shouldn't there be 2 distinct features?